AnimatedTextInput/Classes/AnimatedTextInput.swift

```diff
@@ -363,7 +363,7 @@ open class AnimatedTextInput: UIControl {
     fileprivate func styleDidChange() {
         lineView.defaultColor = style.lineInactiveColor
         placeholderLayer.foregroundColor = style.placeholderInactiveColor.cgColor
-        let fontSize = style.textInputFont.pointSize
+        let fontSize = style.placeHolderFont.pointSize
         placeholderLayer.fontSize = fontSize
         placeholderLayer.font = style.textInputFont
```

Comment:
Please review your code, you missed this one:

`placeholderLayer.font = style.placeHolderFont`

Reply:
Also change it here:
https://github.com/jobandtalent/AnimatedTextInput/pull/91/files#diff-10107ec561d677b4fb147c90c7d572f4L235